qt/scientific_interfaces/ISISReflectometry/GUI/Runs/RunsPresenter.cpp

```diff
@@ -186,10 +192,17 @@ void RunsPresenter::reductionPaused() {
 /** Resume autoreduction. Clears any existing table data first and then
  * starts a search to check if there are new runs.
  */
-bool RunsPresenter::resumeAutoreduction() {
-  auto const searchString = m_view->getSearchString();
+bool RunsPresenter::resumeAutoreduction(std::string searchString) {
```

Comment:
Why do we now pass the search string in here? Is it just for testing? If so then we can get the mock view to return the test value we want instead.

Reply:
It seems so. I will work on mocking for the testing.